source/mainnet/net/guides/shield-ccd-wallets.rst

```diff
@@ -1,28 +1,42 @@
 .. include:: ../../variables.rst
 .. _shield-CCD-wallets:
 
-=====================================
-Shield and unshield CCD on an account
-=====================================
+==========================
+Unshield CCD on an account
+==========================
 
-Accounts on the Concordium blockchain have two balances, the **Balance** and the :term:`shielded balance`. You can move funds between these
-two balances using either a :term:`shield CCD transaction<shielding>` or an :term:`unshield CCD transaction<unshielding>`.
+.. _shielded-balance-feature-deprecation:
 
-When you shield an amount on an account, only the account's credential holder can see the shielded amounts. Other participants in the network will be able to see the shielding transaction, but can't see the shielded balance or any shielded transfers going in or out of the account. You can't make shielded transfers on multi-signature accounts, only on accounts with a single credential.
+.. Warning::
 
-If you have an account with CCD in the shielded balance, you can't add new credentials to that account. Furthermore, once a shielded transfer has been logged on an account, you'll no longer be able to add more credentials to that account. This is also the case even if you unshield all CCD in the shielded balance.
+   Functionalities related to shielding a balance are deprecated in protocol 7 and above.
+   No additional shielded balance can be added to an account and no transfer of shielded balance is possible.
+   Only unshielding of an already shielded balance is possible and recommended to be done.
+   Wallets and command-line tools will continue to display shielded balances and support the
+   unshielding flow to recover already shielded funds.
 
 .. Warning::
-   Do not shield all of your funds or you will not have enough funds to cover transaction fees for unshielding or sending them.
+    This box contains some old context regarding the old shielding feature before protocol 7.
```

Comment:
I would prefer not to have the warning box here, I don't think it looks good when the [entire page content is inside boxes](https://developer.concordium.software/pr-preview/pr-1032/en/mainnet/net/guides/shield-ccd-wallets.html). But of course this is subjective.